### DISCUSSION TITLE
Globally disable git-delta and use only builtin preview

### DISCUSSION THREAD

**Discussion by yavorski** [ibhagwan/fzf-lua#1439] [Q&A] [answered]
Hello, I'm looking to globally disable git-delta for everything - is that possible? 
I'd also like to use only the "builtin" preview for everything across the board - if this is possible too?

The main reason is that I work on both Linux and Windows, and I've noticed that external processes on Windows, especially in Neovim/PowerShell, tends to be quite slow. 
Everything runs smoothly on Linux, but Windows can be a bit of a pain.

I'm thinking of handling this conditionally by OS detection later.

I actually have bat & git-delta installed and I am using them in terminal (so they exist in my path), but I don't want to use them in fzf-lua (for the reasons mentioned above).

Hope this makes sense.
Thanks in advance!

**Comment by ibhagwan** [comment, marked as answer]
> Hello, I'm looking to globally disable git-delta for everything - is that possible?

```lua
require'fzf-lua'.setup({
  defaults = { preview_pager = false },
  -- rest of your settings
})
```

> I'd also like to use only the "builtin" preview for everything across the board - if this is possible too?

That happens already by default unless there is no builtin previewer available (for example, git commits, etc).

> The main reason is that I work on both Linux and Windows, and I've noticed that external processes on Windows, especially in Neovim/PowerShell, tends to be quite slow.
Everything runs smoothly on Linux, but Windows can be a bit of a pain.

Totally understand, opening new processes on Windows isn’t efficient at the OS level, funny enough running WSL on a windows machine is faster than native, if you can do that I highly recommend it.

**Reply by yavorski** [reply]
Thank you so much! 
I can't believe I missed the `defaults` key in the settings - I must have gone through the docs for at least two, three hours today.

**Reply by ibhagwan** [reply]
No worries, it’s in OPTIONS.md btw 
https://github.com/ibhagwan/fzf-lua/blob/main/OPTIONS.md#general-usage
